internal/kldutils/cmdutils.go

```diff
@@ -48,3 +51,17 @@ func DefInt(envVarName string, defValue int) int {
 	}
 	return int(parsedInt)
 }
+
+// MarshalToYAML marshals a JSON annotated structure into YAML, by first going to JSON
+func MarshalToYAML(conf interface{}) (yamlBytes []byte, err error) {
```

Comment:
I'm betting this becomes a hot piece of code at some point, but it gets the job done. Something to keep in the back of our heads to improve performance.

Reply:
It's only used at the moment for the `-Y` parameter, as a helper to dump the output.

There is similar code in the YAML->JSON part of the Webhooks bridge, which from much reading online seems to be the 'correct' way to do things with the inability to tag up a structure for _both_ YAML and JSON. The performance fix for users there is to submit JSON directly.

YAML's really just for humans using Postman/Curl to try things out.